src/filter/owner.rs

```diff
@@ -0,0 +1,113 @@
+use anyhow::{anyhow, Result};
+use std::fs;
+
+#[derive(Clone, Copy, Debug, PartialEq)]
+pub struct OwnerFilter {
+    uid: Check<u32>,
+    gid: Check<u32>,
+}
+
+#[derive(Clone, Copy, Debug, PartialEq)]
+enum Check<T> {
+    Equal(T),
+    NotEq(T),
+    Ignore,
+}
+
+impl OwnerFilter {
+    pub fn from_string(input: &str) -> Result<Self> {
+        let mut it = input.split(':');
+        let (fst, snd) = (it.next(), it.next());
```

Comment:
Could we add a check here to make sure that `it.next()` is `None` after calling it twice? To disallow usage of two or more colons like `--owner shark:lxd:whatever` which would likely be a mistake.

Reply:
Yes, seems like a good thing to check. I'll add it.